test/Directory.Build.props

```diff
@@ -0,0 +1,14 @@
+﻿<Project>
+  <Import Project="../Directory.Build.props" />
+
+  <!-- Build configuration -->
+  <PropertyGroup>
+    <TargetFrameworks>net452;netcoreapp2.2</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT' OR '$(CoreOnly)' == 'True'">netcoreapp2.2</TargetFrameworks>
+  </PropertyGroup>
+
+  <!-- Debug configuration -->
+  <PropertyGroup>
+    <DebugType>portable</DebugType>
```

Comment:
Does this belong in the root Directory.Build.props so it can affect src as well?

Reply:
It's for test libraries only. So no, it doesn't affect.

Reply:
IIRC this property means that portable PDBs are generated rather than the classic ones - I think this is desirable also for the Npgsql project itself. Unless I'm mistaken we include these PDBs in our nuget too.

Reply:
Will check, but previously we hadn't that property in the main project.

Reply:
That's true - I remember thinking about it, but it was a bit too early at the time, and not all tooling supported it. But for the next version it really should be fine.

Reply:
Moved it to the root props.